packages/components/icons/README.md

```diff
@@ -42,3 +42,11 @@ Main use cases are:
     label={this.props.intl.formatMessage(messages.exportList)}
   />
   ```
+
+## Adding New Icons to UI-Kit
+
+1. Add the icon's SVG to `packages/components/icons/src/raw-components` directory. Please follow the style conventions of existing icon files. File names are the icon's name and are kebab-case.
```

Comment:
Might read better as `File names should be the icon's name in kebab-case` + an example